env-defaults

```diff
@@ -269,6 +269,13 @@ QUERY_CMR_REQUESTS_MEMORY=128Mi
 QUERY_CMR_LIMITS_CPU=128m
 QUERY_CMR_LIMITS_MEMORY=512Mi
 
+GIOVANNI_ADAPTER_IMAGE=harmonyservices/giovanni-adapter:latest
```

Comment:
Note that we're going to need to add the Giovanni service setup in harmony-ci-cd in order to start using it in AWS deployed environments. Are you doing this with a separate ticket?

Reply:
Yes, I remember Patrick suggesting doing it later ... can you confirm @bilts ?